src/modules/organisation-role/organisation-role.controller.ts

```diff
@@ -89,6 +94,18 @@ export class OrganisationRoleController {
     }
   }
 
+  @Delete(':id/roles:roleId')
```

Comment:
This path looks incorrect.

```suggestion
@Delete(':id/roles/:roleId')
```

Reply:
I will fix this.